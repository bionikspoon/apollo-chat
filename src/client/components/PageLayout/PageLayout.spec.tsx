import { mount } from 'enzyme'
import * as React from 'react'
import PageLaout from '.'

test('it matches snapshot', () => {
  const wrapper = mount(
    <PageLaout>
      <h1>Test</h1>
    </PageLaout>
  )
  expect(wrapper).toMatchSnapshot()
})
