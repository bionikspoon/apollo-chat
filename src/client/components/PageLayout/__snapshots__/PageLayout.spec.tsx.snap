// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
<PageLayout>
  <div
    className="bp3-ui-text bp3-dark container_ybok0l"
  >
    <h1>
      Test
    </h1>
  </div>
</PageLayout>
`;
